valueformatting_test.go

```diff
@@ -520,6 +520,42 @@ func TestValueFormattingFormat(t *testing.T) {
 	}
 }
 
+func TestProtobufferAndYAML(t *testing.T) {
+
+	globalValueFormatting = newValueFormatting()
```

Comment:
nit: again I see the convention is the longer names, but `vf`, or `f` might be more Go-like here.

Reply:
Good point. However, this is a global variable declared in the valueformatting.go file, so I'm inclined to leave it as-is.